src/react/components/icons/Info.jsx

```diff
@@ -24,12 +44,14 @@ const StyledInfoIcon = styled(QuestionCircleOutlined)`
 export const Info = ({ children, title, size = 16, style }) => {
 
     return (
-        <Tooltip title={title || children}>
-            <StyledInfoIcon
-                className='t_icon t_info'
-                style={{ fontSize: `${size}px`, ...style }}
+        <ThemeProvider>
```

Comment:
ThemeProvider should not be in the icon but somewhere where this is used. The theme might be different if the usage is on a map plugin or in a geoportal setting.